lucien-ios/LucienConstants.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  LucienConstants.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/3/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+final class LucienConstants {
+    static let keyBoardHieghtPadding = CGFloat(50)
```

Comment:
Does this need to be shared between classes or does it only apply to `AddComicBook`? Also watch out for typos

Reply:
Yes. I will need when working with the edit comic form.

Reply:
I'll fix the typos.